docs/content/components/list.mdx

```diff
@@ -12,6 +12,12 @@ import { Wrapper } from '../../utils/Wrapper';
 
 # List
 
+- At its core, the `List` component allows you to easily display a related
+  content grouped together and organized vertically.
+- Generally, every `List` should be used to present simple pieces of
+  information. For more complex sets of data, consider using a Data Table.
+- A `List` can be Ordered and Unordered. See Variants.
```

Comment:
@artsheeps Sometimes the definitions are bullet points and sometimes paragraphs. Is it worth making them consistent?

Reply:
Yes! On the commit I was working ages ago I had this improvement! I will try to work around the conflicts and fix this!